tests/main/selinux-lxd/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Check that LXD snap works on a system using SELinux in enforcing mode
+
+description: |
+    Make sure that LXD snap can be installed and used on a SELinux enable
+    system, with enforcing mode on.
+
+# Systems with SELinux enabled out of the box.
+systems: [fedora-*, centos-*]
+
+# lxd downloads can be quite slow
+kill-timeout: 25m
```

Comment:
Do we really need this (to be that large)? FWTW tests/regression/lp-1815869 installs lxd snap with no kill-timeout defined.

Reply:
Took the timeout and priority values from here: https://github.com/snapcore/snapd/blob/master/tests/main/lxd/task.yaml#L12-L16

Reply:
Hmmm, ok.